### ISSUE TITLE
Allow adding of caption/alt text in media content type

### ISSUE THREAD

**Issue by andrewmee** [strapi/strapi#2133]
It should be possible to provide a string of text for each media file that is added - this could be used as a formal image caption, or to provide the alt text for that image.

**What is the expected behavior?**

In the Admin interface when an image is added, below its thumbnail would be a text field for an editor to type in some text describing the image.

**Comment by lauriejim** [comment]
Hello @andrewmee, thank you for this feedback.

You can custom the plugin upload to add this feature.
You can add a `description` text field in the model and add the new param in the request. (Not completely sure if other updates are needed but it can work I think)

**Comment by Aurelsicoko** [comment]
Sure! We will do it when we'll develop the Media Library plugin. I'm closing the issue to keep the repository as clear as possible. Thank you again for pointing this out!
